frontend/src/components/worksheets/BundleDetail/BundleDetailSideBar.jsx

```diff
@@ -169,7 +169,7 @@ class BundleDetailSideBar extends React.Component<{
                             </div>
                         </CopyToClipboard>
                         <div style={{ flexWrap: 'wrap', flexShrink: 1 }}>
-                            <pre>{bundleInfo.command} </pre>
+                            <pre style={{ whiteSpace: 'pre-wrap' }}>{bundleInfo.command} </pre>
```

Comment:
Inline style is not the best but the file is using withStyles which is also deprecated, so for the scope of this fix we just put it here. This file should be refactored when we does #3883